Makefile

```diff
@@ -72,71 +72,61 @@ essr: VERSION
 
 # Create .tgz and .zip files only
 # GNUTAR=gtar make tarballs
-tarballs: $(ESSDIR)
+.PHONY: rel-tarballs
+rel-tarballs: rel-staging
 	@echo "**********************************************************"
-	@echo "** Making distribution of ESS for release $(ESSVERSION) from $(ESSDIR)/"
-	@echo "** Making pdf and html documentation"
-#	the making of pdf, html, ESSR.rds shall go into `make dist' #752
-	@cd $(ESSDIR)/doc/ ; $(MAKE) pdf
-	@cd $(ESSDIR)/doc/ ; $(MAKE) html
-	@echo "** Creating .tgz file **"
+	@echo "** Making distribution of ESS for (pre)release $(ESSVERSION) from $(ESSDIR)/"
 	test -f $(ESSDIR).tgz && rm -rf $(ESSDIR).tgz || true
+	@echo "** Creating .tgz file **"
 	$(GNUTAR) hcvofz $(ESSDIR).tgz $(ESSDIR)
 	@echo "Signing tgz file"
+ifeq ($(TRAVIS),)
```

Comment:
We use `CONTINUOUS_INTEGRATION` elsewhere for this sort of thing, might as well use it here for consistency.

Reply:
done